_includes/table_widget.html

```diff
@@ -0,0 +1,33 @@
+
+<div id="table-main">
+ <div >Download table as: [ <a href="/registry/ontologies.yml">YAML</a> | <a href="/registry/ontologies.jsonld">JSON-LD</a> | <a href="/registry/ontologies.ttl">RDF/Turtle</a>  ]
+</div>
+    <div style="padding-bottom: 10px; padding-top: 20px;"><input type="text" style="width: 60%;" class="form-control" id="searchVal" placeholder="Search table ..."></div>
+
+    <div class="row">
+        <div class="col-md-3">
+             Ontology Domains:
```

Comment:
Increase accessibility by making an explicit label - that way screen readers can communicate what the select list does
```suggestion
             <label for="dd-domains>Ontology Domains:</label>
```